wiki/markdown/docker.md

```diff
@@ -79,3 +79,34 @@ Another option to configure Eiffel-Intelligence is to provide the application pr
 
 `docker run -p 8070:8080 --expose 8080 --volume /path/to/application.properties:/tmp/application.properties -e spring.config.location=/tmp/application.properties eiffel-intelligence:0.0.19`
 
+
+# Run Docker image with provided docker-compose file
+This docker-compose file includes these components, src/main/docker/docker-compose.yml:
+- MongoDb
+- RabbitMq
+- ER
```

Comment:
I was thinking maybe we need to have a description of the eco system which EI requires to run ... but it's not neccesary in this PR, just a thought that came to mind

Reply:
ok